api/cas-server-core-api-configuration-model/src/main/java/org/apereo/cas/configuration/model/support/pac4j/oidc/Pac4jAzureOidcClientProperties.java

```diff
@@ -18,7 +18,10 @@
     private static final long serialVersionUID = 1259382317533639638L;
 
     /**
-     * Azure AD tenant name.
+     * Azure AD tenant name, it can take 4 different values: 
+     * `common`, `organizations`, `consumers` or specific tenant domain name or ID
```

Comment:
Back-ticks won't work with Javadoc comments. You should be using `{@code }` tags.

Reply:
Also you may want to consider using <ul> and <li> tags to list possible values. It will render better as HTML when exported.